test/runTests.ts

```diff
@@ -11,18 +11,11 @@ import { runTests } from "@vscode/test-electron";
     // Passed to --extensionTestsPath
     const extensionTestsPath = path.resolve(__dirname, "integration", "index");
 
-    const folder = path.resolve(
-      __dirname,
-      "..",
-      "..",
-      "test",
-      "integration",
-      "tests"
-    );
+    const folder = path.resolve(__dirname, "..", "..", "test", "integration");
 
     // Download VS Code, unzip it and run the integration test
     await runTests({
-      version: "1.65.0",
```

Comment:
We should bump the version to `1.70.2` in the root `package.json` and client. Just so they all match.